src/shared/user-record-show.c

```diff
@@ -124,15 +124,34 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
         case -ESTALE:
                 printf(" Password OK: %slast password change in future%s\n", ansi_highlight_yellow(), ansi_normal());
                 break;
-
```

Comment:
why drop this?

Reply:
No idea actually, I'm putting the blank line back ^^